content/post/2023/captioning-tools-extravaganza.md

```diff
@@ -0,0 +1,127 @@
+
+The biggest *downside*, in my experience, with human captioners, is there can be more of a delay between someone speaking and the captions showing up. With the best captioners I've worked with, it's only a couple of seconds - even so, automatic captions tend to be even faster. For a big presentation, this isn't quite as much of a concern; I find delay more frustrating when I'm in a meeting with a fast-moving conversation.
+
+Overall, my personal opinion is that auto-captions work well for me, and that they're a reasonable middle of the road option if a conference doesn't have the budget to offer something better, or for some reason isn't able to organize something better. However, I always advocate listening to the feedback you get from your attendees and, to the greatest extent possible, providing what they've stated they prefer.
+
+### General Thoughts
+
+First, I'd like to call out my operating context here: I'm a Midwestern American white male, working on teams primarily composed of the same. This, unfortunately, is the Platonic ideal for automatic speech to text processing. Depending on your accent, or your teammates', you may have radically different experiences than mine.
+
+Some tools are able to both transcribe audio **and** translate it to another language at the same time. However, as I've never used this capability, I'm unable to say how well it works.
+
+It's also important to emphasize that these tools are not perfect, and sometimes misunderstand what you say - this is pretty common for me because I have a strong "deaf" accent and don't sound like most other people (funnily, I'm occasionally asked if I'm British, so I guess that's how my accent comes across to American ears). One reason I mention this is because many of these tools offer profanity filters, and I've occasionally had them think I was swearing when I wasn't. This is something I wanted to call out; from a speaker's standpoint, if you're not accustomed to this and aren't expecting it, or expecting the audience to react to it, it might be a little surprising.
+
+> A conference I regularly attend - Stir Trek in Columbus, Ohio - runs a series of repeating announcements on the projector screens before each talk. Including a brief educational slide about automatic captions might be a _great_ idea here.
+
+With all of that said, let's examine options across various platforms. I will note if options are free or paid, but since prices are always changing, I'm not going to make an effort to call those out specifically.
+
+### As a Presenter, I want captioning, so that my talk is accessible
+
+Generally, the usual advice for speaking applies. Speak as clearly as you can, avoid cross-talk if you're co-presenting with someone. If you're doing a Q&A session, it's helpful to repeat the question back to make sure it shows up in the captioning feed.
```

Comment:
Here the audience seems to be speakers at conferences. Since it's less likely that they read this than conference organizers, I'd ask conference organizers to ask speakers to speak clearly, etc., so the captioning tool works well and their talk is accessible.

Reply:
I think I'm going to leave this here, but add a note for conference organizers to that effect as well - it's a good point.

I'm also hoping conferences might point their speakers to this as a resource. :)